var/spack/repos/builtin/packages/xsdk/package.py

```diff
@@ -105,16 +106,25 @@ class Xsdk(BundlePackage, CudaPackage, ROCmPackage):
     variant("heffte", default=True, description="Enable heffte package build")
     variant("slate", default=(sys.platform != "darwin"), description="Enable slate package build")
     variant("arborx", default=True, description="Enable ArborX build")
-    variant("exago", default=True, description="Enable exago build")
+    variant("exago", default=True, when="@:1.0.0", description="Enable exago build")
     variant("hiop", default=True, description="Enable hiop build")
     variant("raja", default=(sys.platform != "darwin"), description="Enable raja for hiop, exago")
-    variant("pflotran", default=True, description="Enable pflotran package build")
+    variant("pflotran", default=True, when="@:1.0.0", description="Enable pflotran package build")
 
+    xsdk_depends_on(
+        "hypre@2.32.0+superlu-dist+shared", when="@1.1.0", cuda_var="cuda", rocm_var="rocm"
+    )
     xsdk_depends_on(
         "hypre@2.30.0+superlu-dist+shared", when="@1.0.0", cuda_var="cuda", rocm_var="rocm"
     )
     xsdk_depends_on("hypre@2.26.0+superlu-dist+shared", when="@0.8.0", cuda_var="cuda")
 
+    xsdk_depends_on(
+        "mfem@4.7.0+shared+mpi+superlu-dist+petsc+sundials+examples+miniapps cxxstd=14 ",
```

Comment:
The option `cxxstd=14` should not be necessary. In fact, due to the `sundials` dependency, the C++ standard will be set to C++14 automatically.

Reply:
Thanks! pushed this change

Reply:
BTW: This issue came up with `mfem+hip` so perhaps `cxxstd=auto` should detect hip dependency and set c++14 automatically?

Reply:
Why do you think using HIP requires C++14? I think C++11 is sufficient.

Reply:
I'll have to recheck builds - but I think some of the hip/rocm includes require c++14 - and builds fail when built with `std=cxx11` compiler flag.

Reply:
Created an issue for this failure at https://github.com/xsdk-project/xsdk-issues/issues/267